contracts/Polygon_SpokePool.sol

```diff
@@ -1,12 +1,13 @@
 // SPDX-License-Identifier: GPL-3.0-only
 pragma solidity ^0.8.0;
 
+import "./SpokePool.sol";
+import "./PolygonTokenBridger.sol";
 import "./interfaces/WETH9.sol";
+import "./SpokePoolInterface.sol";
```

Comment:
I think you can remove SpokePoolInterface

Reply:
and in the Arbitrum one!